src/frontend/sharedComponents/GPSPill.tsx

```diff
@@ -37,7 +33,7 @@ export const GPSPill: FC<GPSPillProps> = ({
       IconToRender = GpsSearchingIcon;
       break;
     case 'good': {
-      textValue = `± ${Math.round(locationStatus.accuracy)} m`;
+      textValue = `± ${Math.abs(Math.round(props.accuracy))} m`;
```

Comment:
although `accuracy` isn't supposed to be negative, i figured it made more sense to take the absolute value than to display something like `± -10 m`. Open to an alternative fix (e.g. throwing an error if negative?)